rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc

```diff
@@ -1,24 +1,37 @@
 include::_attributes/attributes-openshift-dedicated.adoc[]
 [id="rosa-shared-vpc-config"]
-= Configuring a shared virtual private cloud for ROSA clusters
+= Configuring a shared VPC for ROSA clusters
 :context: rosa-shared-vpc-config
 
 toc::[]
 
-You can create {product-title} clusters in shared, centrally-managed AWS virtual private clouds (VPCs). This process requires two separate AWS accounts that belong to the same AWS organization. One account functions as the VPC owner while the other account creates the cluster in this VPC.
+You can create {product-title} 
+ifdef::openshift-rosa[]
+(ROSA)
+endif::openshift-rosa[]
+clusters in shared, centrally-managed AWS virtual private clouds (VPCs). 
 
-.Prerequisites
-* You installed the ROSA CLI (`rosa`) 1.2.26 or later.
-* You created all of the required ROSA account roles for creating a cluster.
+[NOTE]
+====
+This process requires *two separate* AWS accounts that belong to the same AWS organization. One account functions as the VPC-owning AWS account (*VPC Owner*), while the other account creates the cluster in the cluster-creating AWS account (*Cluster Creator*).
+====
+
+.Prerequisites for the *VPC Owner*
 * You have an AWS account with the proper permissions to create roles and share resources.
-* You are using an AWS account to create your cluster ("*cluster-creating AWS account*") that is separate from the AWS account that creates your VPC ("*VPC-owning AWS account*").
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
 * Both AWS accounts belong to the same AWS organization.
 * You enabled resource sharing from the management account for your organization.
 * You have access to the link:https://signin.aws.amazon.com[AWS console].
 
+.Prerequisites for the *Cluster Creator*
+* You installed the ROSA CLI (`rosa`) 1.2.26 or later.
+* You created all of the required ROSA account roles for creating a cluster.
```

Comment:
Considering this is an assembly, can we link to https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html#rosa-sts-creating-account-wide-sts-roles-and-policies_rosa-sts-creating-a-cluster-quickly (or something similar) in order to explain _how_ to create the Account Roles?

Reply:
No problem!